moped-editor/src/utils/activityLogFormatters/mopedProjectMigrationActivity.js

```diff
@@ -0,0 +1,9 @@
+import SwitchAccessShortcutAddOutlinedIcon from '@mui/icons-material/SwitchAccessShortcutAddOutlined';
+
+export const formatMigratedProjectActivity = () => {
+  const changeIcon = <SwitchAccessShortcutAddOutlinedIcon />;
+  return {
+    changeIcon,
+    changeText: [{ text: "Imported this project from the legacy project database", style: null }],
+  };
```

Comment:
any suggestions for a better icon or text?

Reply:
I like this icon and feel like it well-suited for the spirit of the migration ✨ I wonder if being more specific about the record coming from the Access DB might be good? Not sure if that would be relevant to anyone.